src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.ts

```diff
@@ -19,13 +19,19 @@ export class NewsListGalleryViewComponent implements OnInit {
 
   ngOnInit() {
     this.textValidationOfMinCharacters();
+    this.textValidationOfMinNameCharacters();
   }
 
   private textValidationOfMinCharacters(): string {
     return this.newsText = (this.ecoNewsModel.text.length >= 198) ?
       ((this.ecoNewsModel.text).slice(0, 197) + '...') : (this.ecoNewsModel.text);
   }
 
+  private textValidationOfMinNameCharacters(): string {
+    return this.ecoNewsModel.author.name = (this.ecoNewsModel.author.name.length >= 11) ?
+      ((this.ecoNewsModel.author.name).slice(0, 11) + '...') : (this.ecoNewsModel.author.name);
```

Comment:
I'm not sure here - is it really important your 'ellipsis'-behavior based on 198/11 symbols?
May you can use native css and based on size of container element?

Reply:
okay, I will use native css